backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc

```diff
@@ -26,10 +26,29 @@ include::modules/oadp-backing-up-pvs-csi.adoc[leveloffset=+1]
 include::modules/oadp-backing-up-applications-restic.adoc[leveloffset=+1]
 include::modules/oadp-using-data-mover-for-csi-snapshots.adoc[leveloffset=+1]
 
+[id="oadp-12-data-mover-ceph"]
+== Using OADP 1.2 Data Mover with Ceph storage
+
+You can use OADP 1.2 Data Mover to backup and restore application data for a cluster that uses CephFS, a cluster that uses CephRBD, and for a cluster that uses both.
+
+OADP 1.2 Data Mover leverages some of the recently added features of Ceph that support large scale environments, one of which is the shallow copy method, which is available for {product-name} 4.12 and later. This feature supports backing up and restoring `StorageClass` and `AccessMode` resources other than what is found on the source PVC.
```

Comment:
is large scale environments, not a compound adjective so should be *large-scale environment* - PEDANTIC - please ignore if you so wish

Reply:
I suggest large-scale environments, since we are speaking of these environments in general.